web/annotation_editor_layer_builder.css

```diff
@@ -55,11 +64,11 @@
 }
 
 .annotationEditorLayer.freeTextEditing {
-  cursor: var(--editorFreeText-editing-cursor);
+  cursor: text;
 }
 
-.annotationEditorLayer.inkEditing {
-  cursor: var(--editorInk-editing-cursor);
+.annotationEditorLayer.freeTextEditing {
+  cursor: var(--editorFreeText-editing-cursor);
```

Comment:
This change doesn't look right, since you're no longer using the ink-cursor anywhere!?